audit/projects/k8s-staging-k8s-gsm-tools/buckets/artifacts.k8s-staging-k8s-gsm-tools.appspot.com/iam.json

```diff
@@ -16,7 +16,14 @@
     },
     {
       "members": [
-        "group:k8s-infra-artifact-admins@kubernetes.io"
+        "group:k8s-infra-staging-k8s-gsm-tools@kubernetes.io"
+      ],
+      "role": "roles/storage.legacyBucketWriter"
+    },
+    {
+      "members": [
+        "group:k8s-infra-artifact-admins@kubernetes.io",
+        "group:k8s-infra-staging-k8s-gsm-tools@kubernetes.io"
```

Comment:
These GCS bucket IAM changes for k8s-staging-gsm-tools are a little surprising to me.

I mean, I know this was a result of running `./ensure-staging-storage.sh` to ensure all staging projects were consistent.  I'm just surprised these weren't here in the first place.

TBH I would like to get rid of the `legacy` bindings if possible